composer.json

```
@@ -21,13 +21,14 @@
 		}
 	],
 	"require": {
-		"php": "~7.4 | ~8.0",
-		"nette/application": "^2.4"
+		"php": "~7.4 | 8.0 - 8.3",
```

Ja bych tady nechal ty původní hodnoty. Tímto říkáme, jakou verzi vyžaduje náš kód v knihovně, neřešil bych podporu ostatních knihoven. Když vyjde php 8.4 tak nejspíš bude náš kód stále kompatibilní. Pokud by závislosti přidaly podporu php 8.4 v minor verzích, tak ani nebudeme muset vydávat novou verzi.

```suggestion
		"php": "~7.4 | ~8.0",
```

---

Ok.